File: poulet-mecanique-backend/src/main/java/fr/arkyan/popo/pouletmecaniquebackend/data/entity/EventProperty.java
package fr.arkyan.popo.pouletmecaniquebackend.data.entity;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Data
public class EventProperty {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column
    private String label;

    @Column
    private String category;

    @Column
    private String discordChannel;

}

File: poulet-mecanique-frontend/src/app/data/event-property.ts
export class EventProperty {

    id: number;
    label: string;
    discordChannel: string;
    category: string;

    constructor(id: number, label: string, discordChannel: string, category: string) {
        this.id = id;
        this.label = label;
        this.discordChannel = discordChannel;
        this.category = category;
    }

}